### ISSUE TITLE
Worldview GLTFScene errored when rendering, (regl) unknown parameter (undefined).

### ISSUE THREAD

**Issue by nutchun** [cruise-automation/webviz#573]
I'm using Worldview in my React project. When I want to render GLTFScene in Worldview component, I got an error like this:
```
Error: (regl) unknown parameter (undefined). possible values: mipmap,nearest mipmap nearest,linear mipmap nearest,nearest mipmap linear,linear mipmap linear,nearest,linear
```
I have used DamagedHelmet model ([https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/DamagedHelmet](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/DamagedHelmet)) to test this out but I got an error when rendering. It seems like the argument `value` in glConstantToRegl from GLTFScene commands received `undefined` value.
Any help for me, thanks in advance.

**Comment by vidaaudrey** [comment]
Looks like we are getting `undefined` texture here since we can't get the correct sampler data from `model.json.sampler`. I uploaded the model to https://gltf-viewer.donmccurdy.com/ and was able to see it. @jtbandes created `GLTFScene` initially, maybe you know what's the best way to add support for this model format?

![image](https://user-images.githubusercontent.com/10999093/103682158-53a73b00-4f3d-11eb-9756-3cc6d4a7fcd4.png)

**Comment by hassoncs** [comment]
@vidaaudrey I think we might need to update `GLTFScene.js` to honor the defaults for the each sampler property. According to this schema I found, it seems like it's valid to have an "empty" sampler which should fall back to the defaults for `magFilter `,`minFilter`,`wrapS`, and `wrapT`.

https://github.com/KhronosGroup/glTF/blob/master/specification/1.0/schema/sampler.schema.json

**Comment by hassoncs** [comment]
Yeah, I think the fix is we should not set those properties (min/mag/wrapS/wrapT) on the `regl.texture` if they aren't present in the `sampler`.

**Comment by vidaaudrey** [comment]
I did try to filter out the undefined texture but end up not seeing the model in the scene. Didn't go deep though. This is different from falling back to default right?

**Comment by hassoncs** [comment]
@vidaaudrey, this was the fix I was thinking of: https://github.com/cruise-automation/webviz/pull/576 I haven't gotten a chance to test it out yet though

**Comment by vidaaudrey** [comment]
@hassoncs  probably still missing something: 
![image](https://user-images.githubusercontent.com/10999093/103965036-afc2b880-5111-11eb-9569-931aefbfb1b6.png)

The https://gltf-viewer.donmccurdy.com/ is showing the model correctly:
![image](https://user-images.githubusercontent.com/10999093/103965098-d84ab280-5111-11eb-8727-75dc359a7920.png)
